src/main/java/racing/NumberGenerator.java

```diff
@@ -0,0 +1,17 @@
+package racing;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class NumberGenerator {
+
+    private final static Random random = new Random();
```

Comment:
```suggestion
    private static final Random RANDOM = new Random();
```

상수 컨벤션은 static final 이고, 변수명은 대문자입니다.